source/_docs/load-and-performance-testing.md

```diff
@@ -1,200 +1,75 @@
-<p>Testing with a session cookie to emulate the experience of a logged-in user is extremely important, as the contrast between an anonymous user and a logged-in user may be drastically different.</p></div>
-
-## Performance Goals
-
-Response times vary from site to site depending on the size of your modules stack, database queries, etc. Generally, anything under 1 second is considered excellent, but this is up to you.
-
-Emulating a logged in user's experience with `ab` is a key metric, as it provides the number of pages per second your site can generate on Pantheon. This number may determine whether or not you need to add additional application containers.
-
-## Testing Tools
-
-There are a number of other tools to consider when you are planning your load testing strategy. This can vary by the need for detail, nature of your site, or requirements for quality analysis.
-
-<table class="table">
-<tbody>
-		<tr>
-			<th>Testing Tool</th>
-			<th>Documentation</th>
-			<th>Acquisition</th>
-		</tr>
-		<tr>
-			<td>Apache AB</td>
-			<td><a href="http://httpd.apache.org/docs/2.2/programs/ab.html">Documentation</a></td>
-			<td><a href="http://httpd.apache.org/download.cgi">Download</a></td>
-		</tr>
-		<tr class="tr_class1">
-			<td>J-Meter</td>
-			<td><a href="http://jmeter.apache.org/usermanual/index.html">Documentation</a></td>
-			<td><a href="http://jmeter.apache.org/download_jmeter.cgi">Download</a></td>
-		</tr>
-		<tr>
-			<td>The Grinder</td>
-			<td><a href="http://grinder.sourceforge.net">Documentation</a></td>
-			<td><a href="http://grinder.sourceforge.net/download.html">Download</a></td>
-		</tr>
-		<tr>
-			<td>Blitz.io</td>
-			<td><a href="http://blitz.io/docs/">Documentation</a></td>
-			<td><a href="https://www.blitz.io/pricing#/subscriptions">Pricing</a></td>
-		</tr>
-	</tbody>
-</table>
+Load and performance tests are critical steps in going live procedures, as they help expose and identify potential performance killers. These tests provide insight for how a site will perform in the wild under peak traffic spikes.
+
+## Load vs Performance Testing
+Before you start, it's important to understand the difference between load and performance testing and know when to use each.
+### Performance Testing
+Performance testing is the process in which you measure an application's response time to proactively expose bottlenecks. These tests should be regularly executed as part of routine maintenance. Additionally, you should run these test before any load testing. If your application is not performing well, then you can be assured that the load test will not go well.  
+
+The scope of performance tests should be limited to the application itself on a development environment (Dev or [Multidev](/docs/multidev)) without caching. This will give you an honest look into your application and show exactly how uncached requests will perform. You can bypass cache by [setting the `no-cache` HTTP headers](/docs/cache-control) in responses.

```

Comment:
My understanding is that the dev environment has a default time-to-live of zero for dev, which implies no caching, but that things that Pantheon Advanced Page Cache may override this to be non-zero value. While a no-cache header may help, this may depend on when this get executed. Suggesting to disable caching via the UI is an option, with an emphasis to remember to re-enable prior to pushing to prod.